app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -52,6 +52,7 @@ protected function validator(array $data)
         return Validator::make($data, [
             'name' => ['required', 'string', 'max:255'],
             'email' => ['required', 'string', 'email', 'max:255', 'unique:users'],
+            'role'=>['required','string'],
```

Comment:
```suggestion
            'role' => ['required', 'string'],
```

Mind the code structure